iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -226,6 +418,18 @@ void populateVMToCPatterns(MLIRContext *context,
   patterns.insert<ConstZeroOpConversion<IREE::VM::ConstI32ZeroOp>>(context);
   patterns.insert<ConstRefZeroOpConversion>(context);
 
+  // List ops
+  // TODO(simon-camp): We leak memory here, as we never release the lists.
```

Comment:
I'm not sure where I want to put the logic. Maybe a cleanup pass under `.../Dialect/VM/Transforms`? Suggestions welcome :)